components/tag/src/tag-add/tag-add.scss

```diff
@@ -16,18 +15,15 @@
     width: 88px;
     font-size: 12px;
     display: inline-flex;
-
     color: $turquoise-700;
     background-color: #ffffff;
     text-align: left;
     vertical-align: middle;
-
     border: 1px solid $turquoise-600;
     border-radius: 12px;
-
-    padding-top: 2px;
-
+    padding: 2px 6px 1px;
```

Comment:
Could you please add a comment here with the reason for a change? Otherwise i will forget it by tomorrow ^^